src/NzbDrone.Core/Notifications/Apprise/Apprise.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using FluentValidation.Results;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Movies;
+
+namespace NzbDrone.Core.Notifications.Apprise
+{
+    public class Apprise : NotificationBase<AppriseSettings>
+    {
+        public override string Name => "Apprise";
+
+        public override string Link => "https://github.com/caronc/apprise";
+
+        private readonly IAppriseProxy _proxy;
+
+        public Apprise(IAppriseProxy proxy)
+        {
+            _proxy = proxy;
+        }
+
+        public override void OnGrab(GrabMessage grabMessage)
+        {
+            _proxy.SendNotification(MOVIE_GRABBED_TITLE, grabMessage.Message, Settings);
+        }
+
+        public override void OnDownload(DownloadMessage message)
+        {
+            _proxy.SendNotification(MOVIE_DOWNLOADED_TITLE, message.Message, Settings);
+        }
+
+        public override void OnMovieAdded(Movie movie)
+        {
+            _proxy.SendNotification(MOVIE_ADDED_TITLE_BRANDED, $"{movie.Title} added to library", Settings);
```

Comment:
`BRANDED` version is used here, but not any of the others. Use either/or on all for consistency.

Reply:
Yep, feel free to fix that one up in another PR if you want.